src/core/plugins/oas3/components/servers.jsx

```diff
@@ -110,7 +110,7 @@ export default class Servers extends React.Component {
               <option
                 value={ server.get("url") }
                 key={ server.get("url") }>
-                { server.get("url") }
+                { server.get("url") } { server.get("description") ? "("+server.get("description")+")" : "" }
```

Comment:
This would be better as a [template literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) 😄

Reply:
Thanks, I will look into that and change it. 👍
I looked in other files beforehand and found similar code style, like [that](https://github.com/swagger-api/swagger-ui/blob/master/src/core/plugins/oas3/components/request-body.jsx#L93).